src/FM.LiveSwitch.Mux/Recording.cs

```diff
@@ -118,7 +118,7 @@ public VideoEvent[] VideoEvents
                         videoEvents.Add(new VideoEvent
                         {
                             Type = VideoEventType.Add,
-                            Timestamp = VideoStartTimestamp.Value,
+                            Timestamp = VideoStartTimestamp ?? DateTime.MinValue,
```

Comment:
I see why you did this, but I'm concerned that this will be unexpected to the calling code.

I think a more semantically consistent approach would be to change this:

```csharp
if (VideoSegments == null)
{
    return null;
}
```

... at line 107 to:

```csharp
if (VideoSegments == null || VideoStartTimestamp == null || VideoStopTimestamp == null)
{
    return null;
}
```

We can then change your approach in `MuxVideo` in `Session` to something that covers a wider range of cases where video segments could not be extracted from a video file for some reason.

We current have:

```csharp
if (recordings.Length == 0)
{
    VideoFile = null;
    _Logger.LogInformation("Session has no video segments.");
    return false;
}
```

... at line 506, and I think:

```csharp
if (recordings.Length == 0)
{
    _Logger.LogInformation("Session has no video segments.");
    return true;
}
```

... makes more sense now. Setting the `VideoFile` to `null` anywhere smells to me now, since it implies that a video file can be orphaned. Same with `AudioFile`.

If we are setting either `AudioFile` or `VideoFile` to `null` to avoid an exception when attempting to delete if it does not exist, it would be better to handle that case in the deletion code.